tareas/clase-5/index4.js

```diff
@@ -0,0 +1,83 @@
+//TAREA: En otro archivo distinto,
+// Crear una lista de <ol> y <li> que contengan sólo números.
+// Convertir esos números a un array y:
+// 1. calcular el promedio y mostrarlo en un <em> pre-creado con el texto "El promedio es..."
+// 2. obtener el número más pequeño y mostrarlo en un <em> pre-creado con el texto "El número más pequeño es..."
+// 3. obtener el número más grande y mostrarlo en un <em> pre-creado con el texto "El número más grande es..."
+// 4. obtener el número que más se repite y mostrarlo en un <em> pre-creado con el texto "El número más frecuente es..."
+
+let $lista = document.querySelectorAll("li");
+let numeros = [];
+
+for (let i =0; i<$lista.length; i++){
+    numeros.push((Number($lista[i].innerHTML)));
```

Comment:
Tenés unos paréntesis de más

```suggestion
    numeros.push(Number($lista[i].innerHTML));
```